plaidml2/bridge/keras/__init__.py

```diff
@@ -1085,11 +1080,13 @@ def softmax(x):
 
 
 def softplus(x):
-    _report_unimplemented('softplus')
+    logger.debug('softplus(x: {})'.format(x))
+    return log(1. + exp(x))
 
 
 def softsign(x):
-    _report_unimplemented('softsign')
+    logger.debug('softsign(x: {})'.format(x))
+    return x / (1. + abs(x))
```

Comment:
This is safe to return like this, but it's sometimes nice to wrap as a KerasNode so you can provide a new name (e.g. 'softsign')